tests/test_sign_with_hsm/get-and-run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2022 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+set -ex
+
+root_directory=$(mktemp -d)
+cd "$root_directory"
+for i in config.sh functions.sh setup.sh test_sign_with_hsm.sh; do
+    curl https://raw.githubusercontent.com/mendersoftware/mender-artifact/master/tests/test_sign_with_hsm/"$i" -o "$i"
```

Comment:
This is very dangerous: a PR modifying the test will never be able to be tested in CI, as this line is getting latest master regardless.

(it is unused, anyway)

Reply:
it was a helper, to run locally. removed.